Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalRequest.h

```diff
@@ -63,21 +44,6 @@ typedef NS_ENUM(NSInteger, BTPayPalRequestUserAction) {
 */
 @interface BTPayPalRequest : NSObject
```

Comment:
question: Is it possible for merchants to instantiate the base class?

Reply:
Yeah. I'm not sure how to make the initializer available to the SDK without it also being available to merchants. It's something I can play around with, though.

Reply:
Yeah it was a thought. Obj-C doesn't have an abstract class concept though this makes sense.

Reply:
Yeah, an abstract class would be perfect here